Sources/WalletConnectSign/Services/App/AppProposeService.swift

```diff
@@ -29,6 +29,7 @@ final class AppProposeService {
         let proposal = SessionProposal(
             relays: [relay],
             proposer: proposer,
+            pairingTopic: pairingTopic,
```

Comment:
SessionProposal does not need this param, we are not relying on protocol object but on pairing topic the proposal has been received on

Reply:
it's not secure to rely on what proposer is claiming here